Bio/SearchIO/ExonerateIO/_base.py

```diff
@@ -106,8 +106,9 @@ def _get_fragments_phase(frags):
 
 
 def _adjust_aa_seq(fraglist):
-    """Transforms three-letter amino acid codes into one-letters in the
-    given HSPFragments.
+    """Transform fragment's 3-letter amino acid codes to one-letter (PRIVATE).
```

Comment:
Probably `'Transform 3-letter AA codes of input fragments to one-letter codes (PRIVATE).'` is a better description.